.cirrus.yml

```diff
@@ -61,6 +61,7 @@ env:
     CTR_FQIN:                # One of the "Container FQIN's" (above)
     CI_DESIRED_DATABASE: sqlite # 'sqlite' or 'boltdb'
     CI_DESIRED_STORAGE: overlay # overlay or vfs
+    ALLOW_LEGACY_DB: false # Required for BoltDB testing now that bolt is deprecated.
```

Comment:
I'm obviously missing something, but I just can't understand what I'm missing.

Why is `$CI_DESIRED_DATABASE` not sufficient?